sdk/cosmos/azure-cosmos-encryption/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,25 @@
+# This is the log4j configuration for benchmarks
```

Comment:
Is it meant for shipping?

Reply:
thanks for catching that. not meant to be there. fixed.